app/controllers/auth/AuthAction.scala

```diff
@@ -36,14 +37,20 @@ class AuthActionImpl @Inject()(override val authConnector: AuthConnector,
                                configuration: Configuration)(implicit ec: ExecutionContext)
   extends AuthAction with AuthorisedFunctions {
 
+  lazy val payeSaAuth: Predicate = ConfidenceLevel.L50 and (Enrolment("IR-SA") or Enrolment("IR-SA-AGENT")) or AuthNino(hasNino = true)
+
+  lazy val saOnlyAuth: Predicate = ConfidenceLevel.L50 and (Enrolment("IR-SA") or Enrolment("IR-SA-AGENT"))
+
+  lazy val auth: Predicate = if(ApplicationConfig.saAuthOnly) saOnlyAuth  else  payeSaAuth
+
```

Comment:
Yes it would. I'm thinking that the separate route could be for SA and PAYE has we have here... Getting rid of the feature switch. It all depends how we plan to use the private beta. I believe we want PAYE and SA users. It also would allow us to play stories where we see ATS reports for both.